src/backend/main/py.main/caliopen_main/user/parameters/user.py

```diff
@@ -27,6 +28,7 @@ class NewUser(Model):
     params = DictType(StringType())
     password = StringType(required=True)
     recovery_email = EmailType(required=True)
+    settings = ModelType(Settings, default=lambda: {})
```

Comment:
`default` could be a func that returns default settings from config file ?

Reply:
default value are a mess and not handled correctly for the moment. default values are defined both in schematics Settings and in swagger object.

I think that I'll remove default value from schematics later and try to see how to handle only API side, ideally using a configuration file.